### PR TITLE
Revisit all xml layout files for Dark Mode audit 

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Revisit all xml layout files for Dark Mode audit:

- Rearrange code in layout files
- Prohibited use of hex color in xml layouts file - added regex check

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

File: scripts/assets/file_content_validation_checks.textproto [hunk 1]
```diff
@@ -341,7 +341,7 @@ file_content_checks {
 }
 file_content_checks {
   file_path_regex: "app/src/main/res/layout.*?/.+?\\.xml"
-  prohibited_content_regex: "@color/(?!component_color_).+"
+  prohibited_content_regex: "@color/(?!component_color_).+|#\\p{XDigit}"
```
Comment (MohitGupta121) [inline scripts/assets/file_content_validation_checks.textproto]: @BenHenning I want to prohibit only Hex value of color, but `#\\p{XDigit}` consider TODO's also such as "TODO(#xxx):" 
So is there any way to we make this as prohibit in regex `android:background="#000000"`  ?
Even we not add extra `"` in regex?

Comment (MohitGupta121) [reply scripts/assets/file_content_validation_checks.textproto]: Thanks @BenHenning, I got the solution by `\"` we can use " in regex.

**Review by MohitGupta121 — COMMENTED** [review]
@BenHenning PTAL, Thanks.
Can I add screenshot for any screen?
As I not change any thing from color side, except to move Hex color of splash activity to component color.

**Review by BenHenning — APPROVED** [review]
Thanks @MohitGupta121. This looks good! I always appreciate a good clean-up. :) Thanks also for adding the extra regex check.

**Comment by BenHenning** [comment]
> @BenHenning PTAL, Thanks. Can I add screenshot for any screen? As I not change any thing from color side, except to move Hex color of splash activity to component color.

@MohitGupta121 given that this is just a simple refactor with no side-effects, a screenshot isn't really needed here.

**Comment by BenHenning** [comment]
Updating PR to latest develop & enabling auto-merge.

**Comment by MohitGupta121** [comment]
Thanks @BenHenning

**Pull request merged**